backstop/tests/menu.html

```diff
@@ -23,22 +39,48 @@
       href="../assets/demo.css"
     />
     <style>
-      #demo-default {
+      #demo-list {
         width: 50%;
       }
+
+      .iui-inline input {
+        width: 218px;
+      }
+
+      #demo-right-align {
```

Comment:
These rules will be needed in react and we try to avoid adding CSS rules there. Why can't we have separate CSS for filters and leaving menu as before?